br/pkg/lightning/backend/local/local.go

```diff
@@ -1689,6 +1689,18 @@ func (local *Backend) GetImportedKVCount(engineUUID uuid.UUID) int64 {
 	return e.importedKVCount.Load()
 }
 
+// GetExternalEngineKVStatistics returns kv statistics of some engine.
+func (local *Backend) GetExternalEngineKVStatistics(engineUUID uuid.UUID) (
+	totalKVSize int64, totalKVCount int64) {
+	v, ok := local.externalEngine[engineUUID]
+	if !ok {
+		// we get it after import, but before clean up, so this should not happen
```

Comment:
I don't understand it. Can you explain more details about the process of local engine import?

Reply:
copied from prev `GetImportedKVCount`, will remove it

Reply:
https://github.com/pingcap/tidb/pull/46998/commits/e25413d647f9ae7b6ac9a8991cd292a828523bab